node_modules/oae-admin/tenants/tenants.html

```diff
@@ -112,6 +122,12 @@
                             </div>
                         </div>
                         <div class="form-group">
+                            <label for="tenants-create-email-domain" class="col-xs-12 col-sm-3 control-label">__MSG__EMAIL_DOMAIN_COLON__</label>
+                            <div class="col-xs-12 col-sm-9">
+                                <input type="text" name="emailDomain" id="tenants-create-email-domain" placeholder="__MSG__EMAIL_DOMAIN__" class="form-control required url"/>
```

Comment:
Oops. I've simply removed the `required` attribute for now, but if there is much more in this area you'd like me to change later, I'd like to hold off on doing anything here until tenant paging is merged, since all that is ported out into a new `createtenant` widget.